webpack/webpack.prod.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const webpack = require('webpack');
+const commonConfig = require('./webpack.config');
+
+function createConfig(affix, format) {
+  const prodConfig = {
+    externals: {
+      react: 'react',
+      'react-dom': 'reactDom',
+      'prop-types': 'propTypes',
+    },
+    mode: 'production',
+    entry: path.resolve(__dirname, './../src/index.js'),
+    output: {
+      path: path.resolve(__dirname, './../dist'),
+      filename: `webexWidgets.${affix}.js`,
+      library: 'WebexWidgets',
+      libraryTarget: format,
+    },
+  };
+
+  return Object.assign(prodConfig, commonConfig);
+}
+
+module.exports = [createConfig('cjs', 'commonjs'), createConfig('umd', 'umd'), createConfig('esm', 'var')];
```

Comment:
I think `var` combines all formats together. not sure which format is only for `esm`.  more info here: https://webpack.js.org/guides/author-libraries/#add-librarytarget

Reply:
> variable: as a global variable made available by a script tag (libraryTarget:'var').

Definitely not the option we're looking for, this would make it so you could do `window.WebexWidgets`.

It looks like webpack doesn't support esm building until webpack v5 according to https://github.com/webpack/webpack/issues/2933

I believe this was the reason we went with rollup for esm in the original widgets.